src/fetch_tse_data.py

```diff
@@ -0,0 +1,254 @@
+import shutil
+import pandas as pd
+import numpy as np
+import os
+import urllib
+import zipfile
+import glob
+
+
+
+FILENAME_PREFIX='consulta_cand_'
+TEMP_PATH = '../data/tse_temp'
```

Comment:
This might fail in Windows (there it's `\` not `/`). We recommend using `os.path.join` to make file and directory paths that would work in both worlds.